Languages/Danish/Keyed/Multiplayer.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>
+
+  <MpSimulating>Simulerer</MpSimulating>
+  <MpSimulatingServer>Simulerer serveren</MpSimulatingServer>
+  <MpLoading>Indlæser</MpLoading>
+  <MpSaving>Gemmer</MpSaving>

```

Comment:
```suggestion
  <MpSaving>Gemme</MpSaving>
```

Reply:
idk is that more correct?

Reply:
"Gemme" would be save and not saving

Reply:
ah ok I don't actually know danish though, I know english, some japanese, and some spanish.